bin/barclamp_lib.rb

```diff
@@ -457,7 +152,8 @@ def opt_parse
 
 def get_user_password
   key = ENV["CROWBAR_KEY"]
-  if key.nil? and ::File.exists?(@crowbar_key_file) and ::File.readable?(@crowbar_key_file)
+
+  if key.nil? and ::File.exist?(@crowbar_key_file) and ::File.readable?(@crowbar_key_file)
```

Comment:
Style/AndOr: Use `&&` instead of `and`. (https://github.com/bbatsov/ruby-style-guide#no-and-or-or)